themes/StarterTheme/templates/catalog/products.tpl

```diff
@@ -1,17 +1,21 @@
-{if $products|count}
-  <section id="products">
-    <h1>{l s='Products'}</h1>
+<section id="products">
+  {if $products|count}
```

Comment:
Still have to get products into a list for the schema on a category listing. [@mfhepp|Dr. Martin Hepp](https://github.com/mfhepp) of good relations explains it best on https://github.com/schemaorg/schemaorg/issues/605